.gitignore

```diff
@@ -1,38 +1,34 @@
-lib
-src/**/*.css.ts
-packages/**/*.css.ts
-styles/**/*.css.ts
-
-node_modules
-npm-debug.log
+# logging / private config
 lerna-debug.log
 yarn-error.log
 .DS_STORE
 .DS_Store
 .chrome
 .firefox
-.npmrc
 .tsbuildinfo
 .rush
 .devcontainer
 tach-results.*
 
+# documentation
 documentation/components/searchIndex.json
+documentation/api-docs
+documentation/dist
 
+# dependency folders
 coverage
+lib
+
+# compiled / pulled-down assets
```

Comment:
What did you have in mind with "pulled-down" here? These are all various build artifacts, IIRC.

Reply:
Ah good, that's what I wasn't sure about - if custom-elements.json for example was a build artifact or something pulled-down externally

Reply:
```suggestion
# build artifacts
```